core-bundle/src/Picker/ArticlePickerProvider.php

```diff
@@ -12,8 +12,25 @@
 
 namespace Contao\CoreBundle\Picker;
 
+use Knp\Menu\FactoryInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Core\Security;
+use Symfony\Component\Translation\TranslatorInterface;
+
 class ArticlePickerProvider extends AbstractInsertTagPickerProvider implements DcaPickerProviderInterface
 {
+    /**
+     * @var Security
+     */
+    private $security;
+
+    public function __construct(Security $security, FactoryInterface $menuFactory, RouterInterface $router, TranslatorInterface $translator = null)
```

Comment:
Same here, `$security` is required in addition to the other required parameters and should therefore be added at the end.